--- a/src/targets/gpu/include/migraphx/gpu/ck.hpp
+++ b/src/targets/gpu/include/migraphx/gpu/ck.hpp
@@ -30,8 +30,9 @@
 #include <migraphx/stringutils.hpp>
 #include <string_view>

-#include "ck/host/device_gemm_multiple_d.hpp"
-#include "ck/host/device_batched_gemm_softmax_gemm.hpp"
+#include <ck/ck.hpp>
+#include <ck/tensor_operation/gpu/device/device_gemm_multiple_d.hpp>
+#include <ck/tensor_operation/gpu/device/device_batched_gemm_softmax_gemm.hpp>

 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
