--- a/src/targets/gpu/CMakeLists.txt
+++ b/src/targets/gpu/CMakeLists.txt
@@ -57,7 +57,7 @@ if(MIGRAPHX_USE_COMPOSABLEKERNEL)
     target_compile_definitions(migraphx_gpu PRIVATE MIGRAPHX_USE_COMPOSABLEKERNEL)
     find_package(
         composable_kernel
-        COMPONENTS jit_library device_other_operations device_gemm_operations
+        COMPONENTS device_other_operations device_gemm_operations
                    device_conv_operations device_reduction_operations REQUIRED)
     target_link_libraries(
         migraphx_gpu
